<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpActivity</class>
 <widget class="QWidget" name="SignUpActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="MyWidgetRegister" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <widget class="QPushButton" name="pushButton_post">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>490</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>531</width>
        <height>361</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>321</height>
       </size>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
     </widget>
     <widget class="QWidget" name="widget_sign_up_nav" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>959</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="font">
          <font>
           <family>隶书</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>活动报名</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>789</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_min">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>--</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_close">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/Login/C:/Users/ASUS/Pictures/Login/wkf.png</normaloff>:/Login/C:/Users/ASUS/Pictures/Login/wkf.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请输入下面的报名信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>90</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请选择想要参加的具体项目</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_child_data">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>321</width>
        <height>361</height>
       </rect>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidgetRegister</class>
   <extends>QWidget</extends>
   <header location="global">mywidgetregister.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
